server/src/db/menu.js

```diff
@@ -0,0 +1,24 @@
+const menu = [
+  {
+    id: 1111,
+    name: "Fried Rice & chicken",
+    price: "$20"
+  },
+  {
+    id: 2222,
+    name: "Pizza",
+    price: "$15"
+  },
+  {
+    id: 2222,
+    name: "Coperate Meals",
+    price: "$70"
+  },
+  {
+    id: 2222,
+    name: "cake delicacy",
+    price: "$50"
```

Comment:
If at any point int your application, you will be aggregating meal costs, you probably want to change this from string to number and only append currently at view level.